### PR TITLE
Table copy switched to view

### PR DESCRIPTION
**Pull request by sdreyer**

### CODE REVIEW

**Review by simba-git — APPROVED** [review]
lgtm

File: ff/serving/provider/sql.go [hunk 1]
```diff
@@ -145,12 +144,19 @@ func (store *sqlOfflineStore) tableExists(id ResourceID) (bool, error) {
 	}
 	query := store.query.tableExists()
 	err = store.db.QueryRow(query, tableName).Scan(&n)
-	if n == 0 {
-		return false, nil
+	if n > 0 && err == nil {
+		return true, nil
 	} else if err != nil {
```
Comment (Sami1309) [inline ff/serving/provider/sql.go:149]: no need for an else if since the previous is a return

File: ff/serving/provider/sql.go [hunk 2]
```diff
@@ -145,12 +144,19 @@ func (store *sqlOfflineStore) tableExists(id ResourceID) (bool, error) {
...
 		return false, err
 	}
-	return true, nil
+	query = store.query.viewExists()
+	err = store.db.QueryRow(query, tableName).Scan(&n)
+	if n > 0 && err == nil {
+		return true, nil
+	} else if err != nil {
+		return false, err
```
Comment (Sami1309) [inline ff/serving/provider/sql.go:157]: Is there no distinction programatically between a table existing but no view and both not existing? Could this prompt the creation of a view if the first is not an error but no rows return and the next is false?

Comment (sdreyer) [reply ff/serving/provider/sql.go:157]: This checks to see if there is an existing table with the name. If no results are returned and there is no error, it then checks to see if there is an existing view with that name. If no results are returned and theres no error, then it returns false. 

Tables and views can be interchageable by the rest of the offline store so only one needs to exist

File: ff/serving/provider/sql.go (hunk 1 above)
Comment (sdreyer) [reply ff/serving/provider/sql.go:149]: yeah could change that to two IFs, but felt like theres more organization with if/elseif

File: ff/serving/provider/sql.go (hunk 2 above)
Comment (Sami1309) [reply ff/serving/provider/sql.go:157]: Ok sounds good.

File: ff/serving/provider/sql.go (hunk 1 above)
Comment (Sami1309) [reply ff/serving/provider/sql.go:149]: ok just a style thing no biggie

**Review by Sami1309 — APPROVED** [review]
lgtm

**Pull request merged**